src/SFML/Audio/SoundStream.cpp

```diff
@@ -78,6 +78,8 @@ void SoundStream::initialize(unsigned int channelCount, unsigned int sampleRate)
 {
     m_channelCount = channelCount;
     m_sampleRate   = sampleRate;
+    m_samplesProcessed = 0;
+    m_isStreaming = false;
 
```

Comment:
Not really necessary, but since the two assignments were equal-sign aligned above, maybe the two new ones could get aligned as well?